packages/govern-create/buidler.config.js

```diff
@@ -90,7 +97,7 @@ task('deploy-govern', 'Deploys an Govern from provided factory')
       factoryAddr
     )
     const tx = await GovernBaseFactory.newDummyGovern(name, {
-      gasLimit: 3500000,
+      gasLimit: 4000000,
```

Comment:
Interesting; did the gas requirements for deploying a DAO go up?

Reply:
This is for deploying the Govern Factory so we are fine. But with the previous value was reverted